tests/integration/test_fim/test_registry/test_registry_report_changes/test_disk_quota/test_registry_disk_quota_bigger_file_limit.py

```diff
@@ -0,0 +1,205 @@
+
+pytest_args:
+    - fim_mode:
+        realtime: Enable real-time monitoring on Linux (using the 'inotify' system calls) and Windows systems.
+        whodata: Implies real-time monitoring but adding the 'who-data' information.
+    - tier:
+        0: Only level 0 tests are performed, they check basic functionalities and are quick to perform.
+        1: Only level 1 tests are performed, they check functionalities of medium complexity.
+        2: Only level 2 tests are performed, they check advanced functionalities and are slow to perform.
+
+tags:
+    - fim_registry_report_changes
+"""
+import os
+
+import pytest
+from wazuh_testing import global_parameters
+from wazuh_testing.fim import (LOG_FILE_PATH, KEY_WOW64_32KEY, KEY_WOW64_64KEY, generate_params,
+                               calculate_registry_diff_paths, registry_value_create, registry_value_update,
+                               registry_value_delete, create_values_content)
+from wazuh_testing.fim_module.fim_variables import (WINDOWS_HKEY_LOCAL_MACHINE, MONITORED_KEY, MONITORED_KEY_2,
+                                                    SIZE_LIMIT_CONFIGURED_VALUE, ERR_MSG_CONTENT_CHANGES_EMPTY,
+                                                    ERR_MSG_CONTENT_CHANGES_NOT_EMPTY)
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools.monitoring import FileMonitor
+
+# Marks
+
+pytestmark = [pytest.mark.win32, pytest.mark.tier(level=1)]
+
+# Variables
+
+test_regs = [os.path.join(WINDOWS_HKEY_LOCAL_MACHINE, MONITORED_KEY), 
+             os.path.join(WINDOWS_HKEY_LOCAL_MACHINE, MONITORED_KEY_2)]
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "data")
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+scan_delay = 10
+
+# Configurations
+
+params, metadata = generate_params(modes=["scheduled"], extra_params={
+                                                        "WINDOWS_REGISTRY_1": test_regs[0],
+                                                        "WINDOWS_REGISTRY_2": test_regs[1],
+                                                        "FILE_SIZE_ENABLED": "yes",
+                                                        "FILE_SIZE_LIMIT": "10KB",
+                                                        "DISK_QUOTA_ENABLED": "yes",
+                                                        "DISK_QUOTA_LIMIT": "1KB",
+                                                        })
+
+configurations_path = os.path.join(test_data_path, "wazuh_registry_report_changes_limits_quota.yaml")
+configurations = load_wazuh_configurations(configurations_path, __name__, params=params, metadata=metadata)
+
+
+
+# Fixtures
+
+
+@pytest.fixture(scope="module", params=configurations)
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+
+
+@pytest.mark.parametrize("size", [(8192), (32768)])
```

Comment:
Please add a comment explaining what these numbers mean. We know it, but this test must be clear to external teams.